Dockerfile

```diff
@@ -20,3 +22,7 @@ RUN gem install rake --version =12.3.0 \
     && gem install kitchen-terraform --version 3.0.0 \
```

Comment:
We don't need kitchen-terraform and test-kitchen any more.

Reply:
Yes but I am afraid that it will fail the test of modules which are still using kitchen-terraform.